app/views/shared/_header_logos.html.haml

```diff
@@ -1,10 +1,21 @@
-#header.row
-  #org_logo.col-md-4.hidden-sm.hidden-xs
-    %a{:href => HEADER_LINK_1}
-      = image_tag "/assets/#{CUSTOM_ASSET_PATH}org_logo_197x57.png", :alt => 'org_logo'
-  #department_title.col-md-4
-    %a{:href => HEADER_LINK_2}
-      = image_tag "/assets/#{CUSTOM_ASSET_PATH}department_name_460_by_60.gif",class: 'header_logo_2', :alt => "department_logo"
-  #institution_logo.col-md-4.hidden-xs
-    %a{:href => HEADER_LINK_3, :target => '_blank'}
-      = image_tag "/assets/#{CUSTOM_ASSET_PATH}institution_logo_136x86.gif", :alt => 'institution_logo'
+#header
```

Comment:
SPARC kept forcing me to use this 4-space tab format for this view and I couldn't figure out why. Even when I changed the tab to 2 spaces and adjusted everything, it would complain at me...